example_app_generator/spec/support/default_preview_path

```diff
@@ -26,6 +26,10 @@ require_file_stub 'config/environment' do
     require "action_controller/railtie"
     require "action_mailer/railtie" unless ENV['NO_ACTION_MAILER']
     require "action_view/railtie"
+    if Rails::VERSION::STRING >= '6'
+      require "action_cable/engine"
+      require "action_mailbox/engine"
```

Comment:
It's this line that's causing the "ActiveRecord is defined but should not be!" message in `no_ar_example_app`.

[`action_mailbox` requires `active_record`](https://github.com/rails/rails/blame/main/actionmailbox/lib/action_mailbox/engine.rb#L6).

🎉

Reply:
Good work.  I think the fix is slightly more complex than just removing that line.

Some work on my machine indicates that these changes may do it - https://github.com/petergoldstein/rspec-rails/pull/5 .  I want to ensure they run green on CI before pushing to this branch.